README.md

```diff
@@ -175,6 +183,39 @@ New-PowerAppManagementApp -ApplicationId $appId # Must be run from a Power Platf
 
 > **Notes**: Only authentication via `CertificateThumbprint` is currently supported. We will also be supporting automated app registration in a later release.
 
+### AAD Conditional Access Policy Exemptions
+The ScubaGear ConfigFilePath command line option allows users to define custom variables in YAML format for use in policy assessments against the AAD baseline.  These custom variables are used to exempt specific user and group exclusions from conditional access policy checks that normally would not pass if exclusions are present.  These parameters support operational use cases for having backup or break glass account exclusions to global user policies without failing best practices.  Any exemptions and their risks should be carefully considered and documented as part of an organization's cybersecurity risk management program process and practices.
```

Comment:
Allows both jSOn and YAML.  But you stated that above.  Can probally just drop here.

Reply:
changes made